packages/fether-react/src/assets/theme/shared/animations/SlideLeftIn.styles.js

```diff
@@ -0,0 +1,16 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+import { keyframes } from 'styled-components';
+
+export const SlideLeftIn = keyframes`
```

Comment:
This one's trickier, it's not a React component. I don't have any idea, so let's leave it here for now.